beta/src/pages/learn/start-a-new-react-project.md

```diff
@@ -1,41 +1,41 @@
 ---
-title: Start a New React Project
+शीर्षक: एक नया React शुरू करें
```

Comment:
We do not translate `title` word here. Its used for rendering in docs. Let's keep it as `title`